app/app.go

```diff
@@ -7,12 +7,14 @@ package app
 import (
 	"errors"
 	"fmt"
-	"github.com/madflojo/hord/config"
-	"github.com/madflojo/hord/databases"
-	"github.com/madflojo/hord/databases/cassandra"
-	"github.com/sirupsen/logrus"
 	"strings"
 	"time"
+
+	"hord/config"
```

Comment:
You'll need to specify the full path or this won't build. When you clone this make sure you clone it into `github.com/madflojo/hord` that way the full path is in your `$GOPATH`

Reply:
but , is it the right way to do it ?

Reply:
yes?

Reply:
was it a question ? :D